src/DynamoCoreWpf/ViewModels/Watch3D/DynamoEffectsManager.cs

```diff
@@ -105,6 +105,26 @@ protected void AddDynamoTechniques()
                         BlendStateDescription = DefaultBlendStateDescriptions.BSAlphaBlend,
                         DepthStencilStateDescription = DefaultDepthStencilDescriptions.DSSDepthLess
                     },
+                        new ShaderPassDescription(DefaultPassNames.MeshOutline)
```

Comment:
extra indentation?

Reply:
I notice this is an extra shader pass description for mesh over that of points and lines in addition to the `DefaultPassNames.EffectOutlineP1`. Could you explain what's the difference between `MeshOutline` and `EffectOutlineP1`?